gallery/src/components/NxTable/NxTableErrorExample.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import {
+  NxTable,
+  NxTableCell,
+  NxTableHead,
+  NxTableRow,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+
+const NxTableLoadingExample = () => {
+  return (
+    <NxTable>
+      <NxTableHead>
+        <NxTableRow>
+          <NxTableCell>Header 1</NxTableCell>
+          <NxTableCell>Header 2</NxTableCell>
+          <NxTableCell>Header 3</NxTableCell>
+          <NxTableCell isNumeric>Header 4</NxTableCell>
+          <NxTableCell>Header 5</NxTableCell>
+        </NxTableRow>
+      </NxTableHead>
+      <tbody>
+        <NxTableRow>
+          <NxTableCell isError colSpan={5}>
```

Comment:
I wonder if `NxTable` could have an interface similar to `LoadWrapper`, where it is given information about when it is loading or in error.  Then it could automatically (and consistently) display the error row and loading row, computing their `colSpan` based on the number of header cells

Reply:
Yeah, I was thinking about this last night and I think it could be done to simplify things.